### ISSUE TITLE
Teemo shrooms broken on new update

### ISSUE THREAD

**Issue by Graydenn** [R3nzTheCodeGOD/R3nzSkin#30]
I was using Ver. 1.5.1 and I was (and still am, on that version) able to change the skin of Teemo's shrooms. On the newest update (and I believe the other new releases) the shroom skin is only changed until the shroom hits the ground, upon which it reverts to the default shroom skin.

This holds consistent throughout my testing and is replicable every time.

**Comment by R3nzTheCodeGOD** [comment]
Should now work correctly.

**Comment by Graydenn** [comment]
Works better now, but still experiencing issues.

For example: In this practice tool match I started the game as Cottontail Teemo and then switched to Beemo. The first shroom I placed was malformed (Cottontail texture applied over Beemo shape?) and the rest started working after that. This was not the case in the old update.

![download](https://user-images.githubusercontent.com/42272058/134592998-dcdb43c5-ae8c-4fcf-9c87-a386772ff165.png)
